docs/pages/workflow/prebuild.mdx

```diff
@@ -77,21 +77,21 @@ The `--clean` flag will delete any existing native directories before generating
 
 When you re-run `npx expo prebuild` without the `--clean` flag, it layers the changes on top of your existing files. This is faster than re-generating from scratch, but it may not produce the same results in some cases. For example, not all config plugins are idempotent — when your project utilizes a lot of "dangerous modifiers" for performing changes like adding regex changes to application code, this can sometimes lead to unexpected behavior. This is why using the `--clean` flag is the safest way to use the prebuild command and it is generally recommended in most cases.
 
-Due to the destructive nature of the flag, you'll be warned to have a clean git status when the `--clean` flag is used. This prompt is optional and will be skipped when encountered in CI.
+Due to the destructive nature of cleaning, you'll be warned to have a clean git status when the `--clean` flag is used. This prompt is optional and will be skipped when encountered in CI.
 
 If you'd like, you can disable the check by enabling the environment variable `EXPO_NO_GIT_STATUS=1`.
 
-The purpose of the prompt is to encourage managed workflow users to add the **android** and **ios** directories to the project's **.gitignore**, ensuring that the project is always managed. However, this can make custom config plugins harder to develop so we haven't introduced any mechanism to enforce this behavior.
+The purpose of the prompt is to encourage users to add the **android** and **ios** directories to the project's **.gitignore**, ensuring that the folders are always generated by prebuild. However, this can make custom config plugins harder to develop so we haven't introduced any mechanism to enforce this behavior.
```

Comment:
```suggestion
The purpose of the prompt is to encourage users to add the **android** and **ios** directories to the project's **.gitignore**. This ensures that these directories are always generated by prebuild. However, this can make custom config plugins harder to develop so we haven't introduced any mechanism to enforce this behavior.
```

Let's split the first sentence to improve its readability a bit. Also, let's use "directories" instead of "folders. for consistency.